components/wrapper/Wrapper.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
we can remove the react import since are not necessary and they are not present in other pages 
just for consistency